index.js

```diff
@@ -55,6 +55,13 @@ const module_mgheber = (msg) => {
 	}
 }
 
+const module_recoding = (msg) => {
+	const recode = /tla3.*(dir)*.*[R|r]ecoding[?]*|[R|r]ecoding[ ]*\?/g
+	if (msg.content.match(recode)) {
+		msg.channel.send ('La');
```

Comment:
`msg.channel.send('La')` instead of `msg.channel.send ('La')`

Reply:
both work ya zrg

Reply:
It's about consistency al mghebber